index.bs

```diff
@@ -298,37 +298,65 @@ the PrivacyCG.
 
 <h3 id="bounce-tracking-mitigations-data-model">Data Model</h3>
 
+<h4 id="bounce-tracking-mitigations-data-model-global">Global Data</h4>
+
 The user agent holds a <dfn>user activation map</dfn> which is a [=map=] of
 [=site=] [=hosts=] to [=moments=].  The [=moments=] represent the most recent
 [=wall clock=] time at which the user activated a top-level document on the
-associated [=host=].
+associated [=site=] [=host=].
 
-The user agent holds a <dfn>candidate bounce tracking map</dfn> which is a
+The user agent holds a <dfn>stateful bounce tracking map</dfn> which is a
 [=map=] of [=site=] [=hosts=] to [=moments=].  The [=moments=] represent the
 first [=wall clock=] time since the last execution of the
-[=bounce tracking timer=] at which a page on the given [=host=] performed
-an action that could indicate bounce tracking took place.  For example,
+[=bounce tracking timer=] at which a page on the given [=site=] [=host=] performed
+an action that could indicate stateful bounce tracking took place.  For example,
 if [=bounce tracking timer=] ran at time X and bounces occurred at times X-1,
 X+1, and X+2, then the map value would be X+1.
 
 Note: Schemeless site is used as the data structure key because by default cookies
 are sent to both `http://` and `https://` pages on the same domain.
 
-Note: Hosts are eagerly removed from the [=candidate bounce tracking map=] when
+Note: Hosts are eagerly removed from the [=stateful bounce tracking map=] when
 a user activation occurs.  This means that a given host can exist in either the
-[=user activation map=] or [=candidate bounce tracking map=], but not both at
+[=user activation map=] or [=stateful bounce tracking map=], but not both at
 the same time.  The maps will have non-overlapping sets of keys.
 
+<h4 id="bounce-tracking-mitigations-data-model-per-tab">Per-Tab Data</h4>
+
+Define <dfn>extended navigation</dfn> as a sequence of navigations joined by client-side redirects.
+
+Each [=top-level traversable=] has an associated
+<dfn for="top-level traversable">bounce tracking record</dfn> which is a
+[=bounce tracking record=] or null. This stores the data relevant
+to bounce tracking enforcement for every [=extended navigation=]. Initially it is null, and it becomes
+non-null when an [=bounce tracking record/initial host=] is detected.
+
+A <dfn>bounce tracking record</dfn> is a [=struct=] whose items are:
+<dl dfn-for="bounce tracking record">
+<dt><dfn>initial host</dfn></dt>
+<dd>A [=site=]'s [=host=]. The origin of the current [=extended navigation=].</dd>
```

Comment:
Using "origin" here is a bit confusing.  Do you mean "site"?  Or do you mean initiator of the navigation?